testing/tests/DevExpress.ui.widgets.dataGrid/focus.tests.js

```diff
@@ -5851,6 +5598,48 @@ QUnit.test("autoNavigateToFocusedRow == false and paging if row focused on curre
     this.pageIndex(1);
     this.clock.tick();
 
+    // assert
+    assert.equal(this.option("focusedRowIndex"), -1, "FocusedRowIndex");
+    assert.equal(this.option("focusedRowKey"), "Ben", "FocusedRowkey");
+});
+
+QUnit.testInActiveWindow("autoNavigateToFocusedRow == false, paging, focusedRow has focus", function(assert) {
```

Comment:
Strange test name. What scenarios are tested here?
Focused Row should be updated after paging if grid is focused and autoNavigateToFocusedRow is false?